community/kernel/src/main/java/org/neo4j/kernel/impl/api/CountsStoreTransactionApplier.java

```diff
@@ -68,4 +74,15 @@ public boolean visitRelationshipCountsCommand( Command.RelationshipCountsCommand
         }
         return false;
     }
+
+    @Override
+    public boolean visitSchemaRuleCommand( SchemaRuleCommand command ) throws IOException
```

Comment:
also as we just talk, would be nice to have guarding assertion that will fail in case when this assumption will brake